raft/status.go

```diff
@@ -47,3 +49,20 @@ func getStatus(r *raft) Status {
 
 	return s
 }
+
+func (s Status) MarshalJSON() ([]byte, error) {
+	j := fmt.Sprintf(`{"id":"%x","term":%d,"vote":"%x","commit":%d, "lead":"%x","raftState":"%s","progress":{`,
```

Comment:
@crawford To be more clear, id is the id of a raft statemcahine. term,commit or progress are the index of the log entries. They are different. We only print id as hex string.